src/components/ModalPopup/Body/BodyCallTaxi.tsx

```diff
@@ -0,0 +1,98 @@
+import ButtonShare from "components/Button/ButtonShare";
+import DottedLine from "components/DottedLine";
+import LinkCallTaxi from "components/Link/LinkCallTaxi";
+
+import theme from "tools/theme";
+
+import LocationOnRoundedIcon from "@mui/icons-material/LocationOnRounded";
+import { ReactComponent as KakaoTaxiLogo } from "static/assets/KakaotTaxiLogo.svg";
+import TmoneyOndaLogo from "static/assets/TmoneyOndaLogo.png";
+import { ReactComponent as UTLogo } from "static/assets/UTLogo.svg";
+
+export type BodyCallTaxiProps = {
+  roomInfo: any; // fixme
```

Comment:
`Room` 사용해도 될 것 같습니다 ~

```suggestion
  roomInfo: Room
```

Reply:
앗 아직 타입이 정의되지 않은게 남아있었군요